packages/data/src/types.d.ts

```diff
@@ -27,6 +30,34 @@ export interface WPDataReduxStoreConfig {
     controls?: WPDataFunctionArray,
 }
 
+export type WPDataAtomicStoreSelector<T> = (...args: any[]) => (props: { get: WPAtomResolver<T> }) => T;
+export type WPDataAtomicStoreAction<T> =  (...args: any[]) => (props: { get: WPAtomResolver<T>, set: WPAtomUpdater<T> }) => void;
+
+export interface WPDataAtomicStoreConfig {
+    rootAtoms: Array< WPAtom<any> | WPAtomFamilyItem<any> >,
+    actions?: { [key:string]: WPDataAtomicStoreAction<any> },
+    selectors?: { [key:string]: WPDataAtomicStoreSelector<any> },
+}
+
 export interface WPDataRegistry {
+    /**
+     * Registers a store.
+     */
     register: ( store: WPDataStore ) => void,
+
+    /**
+     * Creates an atom that can be used to subscribe to a selector.
+     */
+    __internalGetSelectorAtom: <T> ( selector: ( get: WPAtomResolver<any> ) => any ) => WPAtom<T>
```

Comment:
Not to myself, check why `__internalGetSelectorAtom` rather than `__internalGetAtomSelector` for consistency.

Reply:
We're creating an "atom" for the "selector" :)  (so it's  not a selector of atoms)